vm-setup/roles/v1aX_integration_test/tasks/verify_provisioned.yml

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Hmm, this can be a bit confusing, since we are checking different resources for different states:

- cluster ==> "provisioned"
- bmh ==> "provisioned"
- m3m ==> "ready"
- machine ==> "runnning"

While I do not have an ideal naming convention for this file, maybe we could rename it to "verify_desired_state.yml" or "verify_resources_states" depnding how do you feel about it.

Reply:
True, I like `verify_resources_states.yml`. Fixing...